docs/presentations/overview.qmd

```diff
@@ -377,16 +377,17 @@ SQL:
 
 - **pandas** (2008): dataframes in Python
```

Comment:
Should we make this section title `Python dataframe history` instead? No strong thoughts.

Reply:
oh I missed that in the redo